src/main/webapp/app/shared/sidebar/conversation-options/conversation-options.component.html

```diff
@@ -0,0 +1,36 @@
+<div class="option-buttons" (click)="$event.stopPropagation()">
+    <button class="favorite btn btn-sm btn-outline-secondary sidebar-button" (click)="onFavoriteClicked($event)" type="button">
+        <fa-icon [icon]="conversation.isFavorite ? faHeartSolid : faHeartRegular" size="sm" />
+    </button>
+    <div ngbDropdown container="body" class="d-inline-block">
+        <button class="btn btn-sm btn-outline-secondary dropdown-toggle sidebar-button" type="button" ngbDropdownToggle>
+            <fa-icon [icon]="faEllipsisVertical" size="sm" />
+        </button>
+        <div ngbDropdownMenu>
```

Comment:
Also this dropdown component should not be tailored to the conversations

Reply:
will be solved in a followup PR